skpro/tests/test_set_output.py

```diff
@@ -0,0 +1,175 @@
+import pandas as pd
```

Comment:
should this not be in the `regression` folder?

Reply:
yes - will move it, should i rename it to something like `test_set_output_regression`?